Lib/fontTools/otlLib/builder.py

```diff
@@ -2763,21 +2763,21 @@ def buildStatTable(ttFont, axes, locations=None, elidedFallbackName=2):
         statTable.AxisValueCount = len(axisValues)
 
 
-def _buildAxisRecords(axes, nameTable):
+def _buildAxisRecords(axes, nameTable, platforms=((1, 0, 0), (3, 1, 0x409))):
```

Comment:
So, the same here, but I think for all the private function (starting with `_`) you should need to bother with default values.